reinforcement_learning/rlclientlib/logger/event_queue.h

```diff
@@ -58,6 +61,18 @@ namespace reinforcement_learning {
       std::unique_lock<std::mutex> mlock(_mutex);
       return _queue.size();
     }
+
+    size_t capacity() const 
+    {
+      return _capacity;
+    }
+
+  private:
+    //thread-unsafe
+    typename queue_t::iterator erase(typename queue_t::iterator it) {
```

Comment:
create an alias for iterator_t to avoid having to use typename in the function signature